ui/package.json

```diff
@@ -12,21 +12,19 @@
   "scripts": {
     "lint": "eslint --ext .ts,.html . --ignore-path .gitignore && prettier \"**/*.ts\" --check --ignore-path .gitignore",
     "format": "eslint --ext .ts,.html . --fix --ignore-path .gitignore && prettier \"**/*.ts\" --write --ignore-path .gitignore",
-    "build": "rimraf dist && tsc && rollup -c rollup.config.js",
+    "build": "rimraf dist && rimraf tsconfig.tsbuildinfo && tsc && rollup -c rollup.config.js",
     "ts-build": "tsc",
     "build:watch": "run-singleton \"tsc --watch --preserveWatchOutput\"",
     "start": "tsc && concurrently -r \"npm run build:watch\" \"wds\"",
     "package": "npm run build && cd dist && bestzip ../dist.zip *"
   },
   "dependencies": {
-    "@holochain-open-dev/cell-client": "^0.8.0",
-    "@holochain-open-dev/utils": "^0.5.6",
-    "@holochain/client": "^0.9.2",
+    "@holochain/client": "^0.11.14",
```

Comment:
You *could*, and it would work, but it's not a best practise. Each package should explicitly specify its own dependencies to work as expected during bundling, publication & installation; and where node ends up putting those folders on disk relative to the workspace is an implementation detail.

Reply:
Reverted this one all good